dhis-2/dhis-api/src/main/java/org/hisp/dhis/period/PeriodType.java

```diff
@@ -646,6 +646,17 @@ public Period createPeriod( String isoDate )
     // CalendarPeriodType
     // -------------------------------------------------------------------------
 
+    public final Period getFuturePeriod( int periodOffset )
+    {
+        Period period = createPeriod();
+
+        // Rewind one as 0 open periods implies current period is locked
```

Comment:
This comment may be out of place here in `PeriodType` to talk about locking. (Perhaps you can explain this better in JavaDoc for the method.)

Reply:
I would if I knew :D I just moved this code.